test/extensions/stats_sinks/metrics_service/metrics_service_integration_test.cc

```diff
@@ -126,7 +125,6 @@ class MetricsServiceIntegrationTest : public Grpc::VersionedGrpcClientIntegratio
     }
     EXPECT_TRUE(known_counter_exists);
     EXPECT_TRUE(known_gauge_exists);
-    EXPECT_TRUE(known_summary_exists);
```

Comment:
This change breaks the gcc build due to write-only local variable:

2020-12-07T22:10:37.7506320Z Execution platform: @rbe_ubuntu_gcc//config:platform
2020-12-07T22:10:37.7507984Z test/extensions/stats_sinks/metrics_service/metrics_service_integration_test.cc: In member function 'testing::AssertionResult Envoy::{anonymous}::MetricsServiceIntegrationTest::waitForMetricsRequest()':
2020-12-07T22:10:37.7509326Z test/extensions/stats_sinks/metrics_service/metrics_service_integration_test.cc:69:10: error: variable 'known_summary_exists' set but not used [-Werror=unused-but-set-variable]
2020-12-07T22:10:37.7510349Z    69 |     bool known_summary_exists = false;
2020-12-07T22:10:37.7510693Z       |          ^~~~~~~~~~~~~~~~~~~~
2020-12-07T22:10:37.7511039Z cc1plus: all warnings being treated as errors

Reply:
🤦 yeah sorry, a commit from earlier in the day didn't make it up here because of a failed git hook that I missed